openedx/features/learner_profile/static/learner_profile/js/views/learner_profile_view.js

```diff
@@ -55,8 +55,11 @@
 
                     HtmlUtils.setHtml(this.$el, HtmlUtils.template(learnerProfileTemplate)({
                         username: self.options.accountSettingsModel.get('username'),
+                        name: self.options.accountSettingsModel.get('name'),
                         ownProfile: self.options.ownProfile,
-                        showFullProfile: self.showFullProfile()
+                        showFullProfile: self.showFullProfile(),
+                        profile_header: gettext('My Profile'),
+                        profile_subheader: gettext('Build out your profile to personalize your identity on edX')
```

Comment:
In a future commit - slightly annoying part about the commit history is that it doesn't update with future commits.